plone/app/contenttypes/indexers.zcml

```diff
@@ -50,5 +50,9 @@
       factory=".indexers.mime_type"
       name="mime_type"
       />
+  <adapter
+      factory=".indexers.getAltTag"
+      name="alt_text"
+      />
```

Comment:
Is this indexer somewhere registered in the catalog?

Reply:
@petschki would that have to happen here? https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/profiles/default/catalog.xml

Reply:
I think so, but that means also an upgradestep in plone.app.upgrades. I'm not sure if we need this indexer at all. `IContentListingObject` looks for the object attribute if no catalog metadata was found. This might be a bit faster for image listings with 1000 items per batch, but TBH that should not be the case in any template at all...

Reply:
... and the catalog doesn't get faster if every information gets indexed in its metadata ... but there might be other oppinions on that 😉 

What I could think of is a `SearchableText` indexer for the alt text maybe?

Reply:
Should be okay to keep without using it.